bash_completion.sh

```diff
@@ -159,14 +166,20 @@ _buildtest ()
       ;;
 
     report|rt)
-      local opts="--end --failure --filter --format --help --helpfilter --helpformat --latest --no-header --oldest --pager --passed --start --terse  -e -f -h -n -p -s -t clear list summary"
-      COMPREPLY=( $( compgen -W "$opts" -- $cur ) )
-      case "${COMP_WORDS[2]}" in summary)
+      local opts="--color --end --failure --filter --format --help --helpcolor --helpfilter --helpformat --latest --no-header --oldest --pager --passed --start --terse  -e -f -h -n -p -s -t clear list summary"     
```

Comment:
can you remove `--helpcolor` from this bash completion this feature is not implemented yet. Please add this in your subsequent PR